src/lib/menu/menu-panel.ts

```diff
@@ -15,7 +15,7 @@ export interface MatMenuPanel {
   yPosition: MenuPositionY;
   overlapTrigger: boolean;
   templateRef: TemplateRef<any>;
-  close: EventEmitter<void | 'click' | 'keydown'>;
+  closed: EventEmitter<void | 'click' | 'keydown'>;
```

Comment:
Presubmit has revealed that this is a problem. Many teams implement `MatMenuPanel`, so changing this to `close` breaks them. This one alone should be changed back to `closed` (aliasing it for the directive)

Reply:
Ok, I switched that interface back to `close` and kept the alias for `close` to `closed` in the directive. Let me know if that is what you were thinking.